### PR TITLE
Feature/feature a

### PR DESCRIPTION
**Pull request by marc-altitude**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,24 +7,22 @@
...
-          font-size: 0;
-        }
+      html,
+      body {
+        width: 100%;
+        height: 100%;
+        padding: 0;
+        margin: 0;
+        overflow: hidden;
+      }
```
Comment (marcpolican) [inline index.html]: ohh good indent yo

File: readme.md [hunk 2]
```diff
@@ -8,3 +8,5 @@ This includes:
 
 Followed this documentation:
 https://doc.babylonjs.com/features/es6_support
+
+Modify readme
```
Comment (marcpolican) [inline readme.md:12]: meh remove this

File: src/index.ts [hunk 3]
```diff
@@ -56,3 +56,5 @@ ground.material = material;
 engine.runRenderLoop(() => {
     scene.render();
 });
+
+// This code is not actually being called lol
```
Comment (marcpolican) [inline src/index.ts:60]: fixed that dude :p

**Review by marcpolican — CHANGES_REQUESTED** [review]
fix these things

File: index.html (hunk 1 above)
Comment (marc-altitude) [reply index.html]: thanks bro

File: readme.md (hunk 2 above)
Comment (marc-altitude) [reply readme.md]: removed

**Comment by marc-altitude** [comment]
done with your requests

**Review by marcpolican — APPROVED** [review]
approved, thanks bro

**Pull request merged**